src/main/java/org/broadinstitute/hellbender/utils/LoggingUtils.java

```diff
@@ -87,6 +93,22 @@ private static void setJavaUtilLoggingLevel(final Log.LogLevel verbosity) {
         return javaUtilLevelNamespaceMap.get(picardLevel);
     }
 
+    /**
+     * Send log output to file if file name is not null.
+     *
+     * @param fileName  File path for log output
+     */
+    public static void sendLogToFile(final String fileName) {
```

Comment:
1. I think it's safer to do the non-null check in sendLogToFile. If the application programmer forgets to do the check, there will  be an error
2. Yes. Added a test.
3. No.